README.md

```diff
@@ -636,7 +636,9 @@ new_price = utils.calculate_price(base_price, voucher, unit_price)
 
 ### Changelog
 
+- **2016-06-08** - `1.0.0` - Release version
 - **2016-05-31** - `0.1.0` - First version:
   - Authentication
   - Voucher informations: *get*, *usage*
   - Voucher operations: *use*
```

Comment:
`use`? chyba raczej `redeem`
poza tym w tej libce jest dużo więcej dostarczone niż to co tutaj podane